src/App.css

```diff
@@ -1,3 +1,337 @@
-.App {
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto+Slab&display=swap');
+
+:root {
+  --Montserrat: 'Montserrat', sans-serif;
+  --RobotoSlab: 'Roboto Slab', serif;
+  --azure: #0290ff;
+  --z: #fafafa;
+  --white: #e8e8e8;
+  --white-two: #fff;
+  --cool-blue: #4386bf;
+  --very-light-pink: #c4c4c4;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  background-color: #f5f6fa;
+}
+
+header {
```

Comment:
- [ ] As mentioned in the previous review, the header which contains the Bookstore text and icon should align with the BookCollection as we can see in the screenshot below
Original Design
![Screenshot from 2023-09-02 06-11-21](https://github.com/SaveryIV/Bookstore-React/assets/46388113/bffe4908-a9dc-487e-be59-1005f6ba4a35)
Your Design
![Screenshot from 2023-09-02 06-07-41](https://github.com/SaveryIV/Bookstore-React/assets/46388113/200ff54b-a93b-4bb3-b9f1-31fa80deccaa)